packages/expo-updates/android/src/main/java/expo/modules/updates/UpdatesDevLauncherController.kt

```diff
@@ -140,6 +141,29 @@ class UpdatesDevLauncherController : UpdatesInterface {
     )
   }
 
+  override fun storedUpdateIdsWithConfiguration(configuration: HashMap<String, Any>?, context: Context?, callback: UpdateCallback?) {
```

Comment:
I'm curious why `configuration`, `context` and `callback` can be nullable here, but not in the `fetchUpdateWithConfiguration` method. It seems we have some default values for them, so it may make sense to make them nullable in both cases 🤔 If not, it may be worth adding a `@Nullable` (https://developer.android.com/reference/androidx/annotation/Nullable) annotation to them in the interface?

Reply:
Good catch, those should not be nullable -- I think Android Studio made them nullable when creating the stub implementation for the interface.